indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -297,6 +310,11 @@
                                    (some?
                                     (some #(= (common-config/cwic-tag) %)
                                           (map :tag-key.lowercase tags))))
+            :granule-data-format granule-data-format
+            :granule-data-format.humanized (-> humanized-values
```

Comment:
`granule-data-format` could have multiple formats, why do we only pick the first humanized value rather than all unique ones?

Reply:
Actually this line can be removed. It is overwritten by `humanized-field->elastic-doc` call anyways.